charts/handbrake/0.0.1/values.yaml

```diff
@@ -0,0 +1,87 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+  strategy:
+    type: Recreate
+
+  service:
+    port:
+      port: 5800            #Required for web GUI
+    additionalServices:
+      - enabled: true
+        nameSuffix: handbrake-vnc
+        type: ClusterIP
+        port:
+          port: 5900        #Optional, only for vnc clients.
+          name: handbrake-vnc
+          protocol: TCP
+          targetPort: 5900
```

Comment:
We do not use additional services, because it fucks up the UI BADLY.

look at some of the Stage-4 apps how to implement more than one service.

Reply:
I'm not familiar with all the apps but zwavejs2mqtt has working example of this

Reply:
I'll check it Thanks!
Is there any additional documentation yet, other than the [wiki](https://github.com/truecharts/truecharts/wiki/TrueNAS-SCALE-Chart-Structure)? 
Because up until now i'm going kinda blindly copy pasting, and testing :D 
Not so much trouble for the simple charts i make, but i'd like to understand why it works also haha !

Reply:
There is no documentation besides the two readme files (see the truenas/charts readme too) actually...
Even our wiki is hugely outdated,.